cmd/kubeadm/app/apis/kubeadm/validation/validation_test.go

```diff
@@ -183,6 +185,30 @@ func TestValidateIPFromString(t *testing.T) {
 	}
 }
 
+func TestValidatePort(t *testing.T) {
+	var tests = []struct {
+		name     string
+		port     int32
+		expected bool
+	}{
+
+		{"negative number port", -1234, false},
+		{"zero number port", 0, false},
+		{"Minimum valid value port", 1, true},
+		{"valid value port", 300, true},
+		{"maximum valid value", 65535, true},
+		{">65535 Invalid value port", 65538, false},
+	}
+	for _, rt := range tests {
+		allErrs := ValidatePort(rt.port, nil)
+		//allErrs[0].Error()
+		if len(allErrs) > 0 {
+			find := strings.Contains(allErrs[0].Error(), "port number is not valid")
+			assert.Equal(t, rt.expected, !find, allErrs[0].Error())
```

Comment:
can we just use regular compare instead of testify?

Reply:
I will modify it according to your opinion，thank you